src/app/models/resources/external/gitlab/repo.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Digital Excellence Copyright (C) 2020 Brend Smits
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU Lesser General Public License as published
+ *   by the Free Software Foundation version 3 of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty
+ *   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *   See the GNU Lesser General Public License for more details.
+ *
+ *   You can find a copy of the GNU Lesser General Public License
+ *   along with this program, in the LICENSE.md file in the root project directory.
+ *   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+ */
+export interface GitLabRepo {
+    id: number;
+    name: string;
+    description: string;
+    readme_url: string;
+    web_url: string;
```

Comment:
Same as for GitHub, why `web_url` instead of simply `url`? Are there plans to support the git URL as well?

Reply:
This is how the GitHub api has structured their resources